code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/ECID.java

```diff
@@ -43,9 +44,9 @@ final class ECID {
 	 */
 	ECID(final String ecidString) {
 		if (StringUtils.isNullOrEmpty(ecidString)) {
-			MobileCore.log(
-				LoggingMode.DEBUG,
+			Log.debug(
 				IdentityConstants.LOG_TAG,
```

Comment:
Let's standardize usage of logs. 

Few files have import static com.adobe.marketing.mobile.edge.identity.IdentityConstants.LOG_TAG;


I prefer` IdentityConstants.LOG_TAG` > `import com.adobe.marketing.mobile.edge.identity.IdentityConstants.LOG_TAG` 

Also, do you think it is better to combine LOG_TAG and LOG_SOURCE like this:
static let LOG_TAG = IdentityConstants.LOG_TAG + ":" + LOG_SOURCE

I am not sure what is the delimiter but have added ":" for the sample.

Reply:
Not sure why this occurrence has a qualified import 😕 (because I did a replace all)! 
I don't mind either way, but if I had to choose I would be inclined towards the static import as it is less verbose (especially when used a lot of time in the class) and also because it is an unambiguous import.